packages/components/src/slot-fill/context.js

```diff
@@ -105,6 +107,10 @@ class SlotFillProvider extends Component {
 		return sortBy( this.fills[ name ], 'occurrence' );
 	}
 
+	hasFills( name ) {
+		return this.fills[ name ] && !! this.fills[ name ].length;
```

Comment:
In a lot of other places, we just check `fills.length`. Is the extra method needed?

Reply:
Can't say for sure, this was a long time ago :P